bin/fixtures/config.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+/**
+ * The configuration of SimpleSAMLphp
+ */
```

Comment:
Would be good to document within this file what came from boilerplate/default and what is different.